pysat/__init__.py

```diff
@@ -82,6 +82,7 @@
     with open(os.path.join(pysat_dir, 'data_path.txt'), 'r') as f:
         data_dir = f.readline()
 
+import netCDF4
```

Comment:
Why is this imported at the init level?

Reply:
Basically to make sure it's loaded before any other package that would load h5py. Newer versions of netCF4 have a custom version of the python HDF5 library that conflicts with h5py.

Reply:
If h5py or a package that imports h5py is loaded before netcdf4 it breaks netcdf4

Reply:
This 'work around' will still break if a packages that uses `h5py` is imported before pysat then?

Reply:
I haven't tested that case, but I would expect so. The real solution is for netCDF4 to resolve their conflict with h5py by using a different namespace or something. In the meantime, we need to either not use newer netCDF4 or use this workaround.

Reply:
Could this be solved by capping the netCDF4 version in setup.py?

Reply:
I believe so, provided that nothing from the newer netCDF4 libraries is needed.

Reply:
I'm not at the VM that I do testing on right now, but I can see what the latest working version of netCDF4 is

Reply:
I just remembered that the latest version of netCDF4 is the only version with a wheel for python 3.8.  This could be an issue for any caps.